README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
-in the Service Manual][service-manual-assistive-technologies].
+- is accessible for users who only use a keyboard
+- lets users adjust or override colours in Windows, Firefox and Chrome
+- is usable in Internet Explorer 8, 9 and 10
 
-Currently, GOV.UK Frontend officially supports the following assistive technologies:
+### Accessibility issues and warnings
 
-| Software                 | Version        | Type               | Browser                   |
-|--------------------------|----------------|--------------------|---------------------------|
-| JAWS                     | 15 or later    | screen reader      | Internet Explorer 11      |
-| ZoomText                 | 10.11 or later | screen magnifier   | Internet Explorer 11      |
-| Dragon NaturallySpeaking | 11 or later    | speech recognition | Internet Explorer 11      |
-| NVDA                     | 2016 or later  | screen reader      | Firefox (latest versions) |
-| VoiceOver                | 7.0 or later   | screen reader      | Safari on iOS10 and OS X  |
+We're working on fixing the following issues from our [issues list in GitHub](https://github.com/alphagov/govuk-frontend/issues/).
 
-In addition, we test that all content is accessible with keyboard only.
+- `legend` elements are not visible to some assistive technologies.
+- Radios with [conditionally revealing content](https://design-system.service.gov.uk/components/radios/#conditionally-revealing-content) fail when they're tested with [Axe](https://github.com/dequelabs/axe-core).
+- The date input component uses `type="number"` and `role="group"`, which causes several accessibility issues.
+- The navigation in the page header uses `aria-label` wrongly, which may cause issues with assistive technologies.
 
-We aim to support [users who adjust or override the colours of websites they visit][how-users-change-colours-on-websites]. We test this by [changing colours in Firefox][changing-colours-in-firefox], by [enabling 'High Contrast' mode in Windows][enabling-high-contrast-mode-in-windows] and by using the [High Contrast plugin for Chrome][high-contrast-plugin-for-chrome].
+Accessibility checkers may also warn you about the following issues. Read about [why these issues should not affect users](https://github.com/alphagov/govuk-frontend/issues/1280).
 
-[service-manual-assistive-technologies]: https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test
-
-[changing-colours-in-firefox]:
-https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use
-
-[enabling-high-contrast-mode-in-windows]:
-https://support.microsoft.com/en-gb/help/13862/windows-use-high-contrast-mode
-
-[high-contrast-plugin-for-chrome]: https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph?hl=en-US
-
-[how-users-change-colours-on-websites]:
-https://accessibility.blog.gov.uk/2017/03/27/how-users-change-colours-on-websites/
+- `aria-disabled` is invalid in disabled elements.
```

Comment:
Again I don't think we should list these out if possible, I'd prefer to keep them in the issue tracker where they'll always be up to date.

We could consider breaking that issue into multiple issues that we then close and give a consistent label.

Reply:
For this second list, it might be a bit odd to have separate GitHub issues for things that aren't issues we'll be working on fixing.

I feel that (unlike the issues-we're-working-on list) this summary list is important, because it helps us meet the user need (essentially "I need to know whether to worry about these warnings") without asking them to read the table / issues in detail. There's still the link if users want to know more, but hopefully some (most?) users can get by with this summary list. What do you think?

Reply:
I'm actually okay with having issues that we won't fix in Github for the record but we can have the `wont-fix` label on them. That way users can find them in Github by searching for them.

Reply:
Ah ok, makes sense - thanks! I'd still like to keep this summary list here though - so we can help as many users as possible with their accessibility tests, without them needing to click and read the full explanation. They can still click through to the `wont-fix` issues list if they'd like the full background.

Reply:
If we're keeping the list in the README then we need to lift the component name from the "component name" column of the [table](https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851).

Because for instance these warnings don't make sense on their own:
```
- A `group` role does not have a name.
- The `contentinfo` role is not necessary.
- The `banner` role is not necessary.
```
I also worry that since we've rephrased the warning message wording, users might not be able to find the error they've seen generated in the README as they'll probably be searching by the exact wording they've seen (that's what I'd do).

Reply:
> For this second list, it might be a bit odd to have separate GitHub issues for things that aren't issues we'll be working on fixing.

Having closed issues with 'wont-fix' attached to them means they're recorded and visible to users (and importantly searchable from Google)